### ISSUE TITLE
cannot install symbol as it is not compatible with VSCode 1.66.2

### ISSUE THREAD

**Issue by biwa1877** [miguelsolorio/vscode-symbols#96]
Hi, I'm trying to install the symbol extension on a company computer which does not have internet access. I downloaded the .vsix file from the marketplace (https://marketplace.visualstudio.com/items?itemName=miguelsolorio.symbols&ssr=false#version-history), and tried to install it in VSCodium. But it said "Unable to install extension 'miguelsolorio.symbols' as it is not compatible with VS Code '1.66.2'." Could you please help? Thanks.

Basically, I'm trying to bring up the outline view in VSCodium for code navigation. Is there any other way of doing so, if symbol does not work? Thanks.

**Comment by miguelsolorio** [comment]
That’s because the theme is only supported on versions 1.70 and higher. I’ll look into seeing if I can scale back the minimum version but I’d highly suggest updating your VS Code to the latest version as yours is more than a year old.
